src/routes/(site)/wave1/+page.svelte

```diff
@@ -8,9 +8,12 @@
   import PositionList from './position-list.svelte';
   import { POSITIONS } from './data';
 
+  let expanded = false;
   function expandAll() {
     const positions = document.querySelectorAll('.position');
-    positions.forEach((el) => el.setAttribute('open', 'true'));
+    if (expanded === true) positions.forEach((el) => el.removeAttribute('open'));
```

Comment:
semantics. 

probably should be one for loop with an `if` inside the block.

Reply:
not gonna fix tho xd123

Reply:
resolved in #842